content/docs/command-reference/commit.md

```diff
@@ -1,7 +1,7 @@
 # commit
 
-Record changes to DVC-tracked files in the <abbr>project</abbr>, by saving them
-to the <abbr>cache</abbr> and updating the `dvc.lock` or `.dvc` files.
+Stores changed changed files and directories tracked by DVC in the
```

Comment:
This way we just put an algorithm here instead of providing some high level summary .. Previous was not ideal, but better to my mind

Reply:
Right. OK sort of rolled back to

> Record changes to files or directories tracked by DVC, by storing the current
versions in the \<abbr\>cache\</abbr\>, and updating `dvc.lock` or `.dvc` files as
needed.

WDYT?

Reply:
just keep `Record changes to files or directories tracked by DVC`  ? (see git commit)

Reply:
Sure, I guess its best to keep the cmd ref intros as short as possible. I moved the other details down to the desc.